navigation.yaml

```diff
@@ -97,6 +97,8 @@ kubernetes:
       path: /kubernetes/install
     - title: Partners
       path: /kubernetes/partners
+    - title: Docs
```

Comment:
lgtm however, if you put this in and merge, it will not allow us to build u.c... perhaps remove this here and then we can?

Reply:
This is targetting a feature branch, Will get  it all ready and set up for merging into master.